src/App.css

```diff
@@ -246,3 +242,100 @@ a.active {
     transform: translateY(0);
   }
 }
+
+/* List & Locations */
+
+.list {
+  min-height: 100%;
+  padding: 0;
+}
+
+.list__locations {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
```

Comment:
`gap` is so awesome, isn't it?

Reply:
The `gap` property is a great 'tool', helpful, and awesome. Congrats to the creators 😄!